include/core/sdirk.h

```diff
@@ -87,5 +87,83 @@ number_of_intermediary_stages(
   return n_stages;
 }
 
+inline unsigned int
+intermediary_stages(Parameters::SimulationControl::TimeSteppingMethod method)
+{
+  if (method == Parameters::SimulationControl::TimeSteppingMethod::sdirk33_1 ||
+      method == Parameters::SimulationControl::TimeSteppingMethod::sdirk22_1)
+    return 0;
+
+  else if (method ==
+             Parameters::SimulationControl::TimeSteppingMethod::sdirk33_2 ||
+           method ==
+             Parameters::SimulationControl::TimeSteppingMethod::sdirk22_2)
+    return 1;
+
+  else // if (method ==
+       // Parameters::SimulationControl::TimeSteppingMethod::sdirk33_3)
+    return 2;
+}
+
+inline unsigned int
```

Comment:
Could also be a const attribute of the class

Reply:
True, but bdf is not a class per say right now. It could become eventually though. Let's leave it like that for the moment :).